react-frontend/src/components/StyleComponents/pageContent.js

```diff
@@ -25,6 +25,26 @@ export const BreadcrumbContainer = styled.div.attrs({
   }
 `;
 
+export const FooterContainer = styled.div.attrs({
+  className: 'footerContainer',
+})`
+  display: flex;
+  flex-direction: column;
+  height: 46px;
+  justify-content: center;
+  text-align: center;
+  @media only screen and (max-width: 900px) {
+    height: 70px;
```

Comment:
using a fixed height makes the footer too tall at some breakpoints:
![image](https://user-images.githubusercontent.com/16946297/97212544-cf3a7e00-177d-11eb-963a-cad3c2078267.png)